[2025-04-14T00:42:12.732-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-14T00:42:12.735-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: daily_weather_pipeline.fetch_weather_dallas scheduled__2025-04-13T00:00:00+00:00 [queued]>
[2025-04-14T00:42:12.737-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: daily_weather_pipeline.fetch_weather_dallas scheduled__2025-04-13T00:00:00+00:00 [queued]>
[2025-04-14T00:42:12.737-0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-14T00:42:12.740-0500] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_weather_dallas> on 2025-04-13 00:00:00+00:00
[2025-04-14T00:42:12.742-0500] {standard_task_runner.py:72} INFO - Started process 78245 to run task
[2025-04-14T00:42:12.744-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'daily_weather_pipeline', 'fetch_weather_dallas', 'scheduled__2025-04-13T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/weather_pipeline_dag.py', '--cfg-path', '/var/folders/4n/q_76d_d57mb9q1qtrf4mvh940000gn/T/tmpnhcdzfxz']
[2025-04-14T00:42:12.745-0500] {standard_task_runner.py:105} INFO - Job 5: Subtask fetch_weather_dallas
[2025-04-14T00:42:12.758-0500] {task_command.py:467} INFO - Running <TaskInstance: daily_weather_pipeline.fetch_weather_dallas scheduled__2025-04-13T00:00:00+00:00 [running]> on host ramyas-macbook-air.local
[2025-04-14T00:42:12.774-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='daily_weather_pipeline' AIRFLOW_CTX_TASK_ID='fetch_weather_dallas' AIRFLOW_CTX_EXECUTION_DATE='2025-04-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-13T00:00:00+00:00'
[2025-04-14T00:42:12.775-0500] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-14T00:42:12.775-0500] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-14T00:42:12.775-0500] {logging_mixin.py:190} INFO - Current task name:fetch_weather_dallas state:running start_date:2025-04-14 05:42:12.735582+00:00
[2025-04-14T00:42:12.776-0500] {logging_mixin.py:190} INFO - Dag name:daily_weather_pipeline and current dag run status:running
[2025-04-14T00:42:12.776-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-14T00:42:36.565-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-14T00:42:36.571-0500] {process_utils.py:132} INFO - Sending 15 to group 78245. PIDs of all processes in the group: [78245]
[2025-04-14T00:42:36.571-0500] {process_utils.py:87} INFO - Sending the signal 15 to group 78245
[2025-04-14T00:42:38.743-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-14T00:42:38.745-0500] {taskinstance.py:2604} INFO - Dependencies not met for <TaskInstance: daily_weather_pipeline.fetch_weather_dallas scheduled__2025-04-13T00:00:00+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2025-04-14T00:42:38.747-0500] {taskinstance.py:2604} INFO - Dependencies not met for <TaskInstance: daily_weather_pipeline.fetch_weather_dallas scheduled__2025-04-13T00:00:00+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2025-04-14T00:42:38.747-0500] {local_task_job_runner.py:166} INFO - Task is not able to be run
